datadog_checks_base/datadog_checks/base/utils/http.py

```diff
@@ -336,11 +335,6 @@ def populate_options(self, options):
 
         return options
 
-    @contextmanager
-    def handle_tls_warning(self):
-        with disable_warnings_ctx(InsecureRequestWarning, disable=True):
```

Comment:
Shall we do the same for other usages of `disable_warnings_ctx` ?

Reply:
We can also get rid of `disable_warnings_ctx` if it's not necessary anymore.

Reply:
done!